src/core/InstrumentFunctions.cpp

```diff
@@ -369,7 +370,7 @@ void InstrumentFunctionArpeggio::processNote( NotePlayHandle * _n )
 
 	const InstrumentFunctionNoteStacking::ChordTable & chord_table = InstrumentFunctionNoteStacking::ChordTable::getInstance();
 	const int cur_chord_size = chord_table[selected_arp].size();
-	const int range = (int)( cur_chord_size * m_arpRangeModel.value() );
+	const int range = (int)( cur_chord_size * m_arpRangeModel.value() * m_arpRepeatsModel.value() );
```

Comment:
```suggestion
	const int range = (int)(cur_chord_size * m_arpRangeModel.value() * m_arpRepeatsModel.value());
```

Reply:
I know it's more typing, but I think we should start using the C++-style casts, too:
```suggestion
	const int range = static_cast<int>(cur_chord_size * m_arpRangeModel.value() * m_arpRepeatsModel.value());
```
Since this is using the C-style cast, it will try `const_cast<int>()` first, then does the `static_cast<int>()` we want.

Reply:
True, I guess it didn't catch my eyes because I used to code in C before learning C++ for LMMS :smile: